app/static/src/app/components/run/RunModelPlot.vue

```diff
@@ -1,8 +1,11 @@
 <template>
-    <div class="run-model-plot" ref="plot" :style="plotStyle">
-    </div>
-    <div v-if="!baseData.length" class="plot-placeholder">
-      {{ placeholderMessage }}
+    <div :style="plotStyle">
+      <div class="run-model-plot" ref="plot">
+      </div>
+      <div v-if="!baseData.length" class="plot-placeholder">
+        {{ placeholderMessage }}
+      </div>
+      <slot></slot>
```

Comment:
Allow container to include additional status message content here, which will be faded along with the plot if required.